OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -155,10 +157,11 @@ void INotifyCreated.Created(Actor self)
 
 		void ITick.Tick(Actor self)
 		{
-			if (!anyCellTouched)
+			if (!anyCellTouched && !disabledChanged)
 				return;
 
 			anyCellTouched = false;
+			disabledChanged = false;
```

Comment:
This line should be moved to the end of the function else the checks will fail

Reply:
Fixed.